src/smart-components/common/catalog-breadcrumbs.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { useSelector } from 'react-redux';
+import ConditionalLink from '../../presentational-components/shared/conditional-link';
+
+const CatalogBreadcrumbs = () => {
+  const fragments = useSelector(
+    ({ breadcrumbsReducer: { fragments } }) => fragments
+  );
+  if (fragments.length <= 1) {
+    return null;
+  }
+
+  return (
+    <Breadcrumb>
+      {fragments.map(({ title, pathname, searchParams }, index) => (
+        <ConditionalLink
+          pathname={!(fragments.length === index + 1) ? pathname : undefined}
+          searchParams={searchParams}
+          nav
+          exact
+          key={pathname}
+          className="pf-c-breadcrumb__item"
```

Comment:
You should check here if it is active, otherwise you dont want to add `cursor: pointer` to it

Reply:
Fixed that with CSS.